src/languages/es-ES.ts

```diff
@@ -589,6 +589,16 @@ export default class extends Language {
 				their marks in a horizontal, vertical, or diagonal row wins the game.`
 		}),
 
+		/**
+		 * ###################
```

Comment:
Always aligned to the title's length:
```suggestion
		 * ################
```